PlantDiary/Pages/Index.cshtml.cs

```diff
@@ -19,7 +19,16 @@ public IndexModel(ILogger<IndexModel> logger)
 
         public void OnGet()
         {
-
+            string brandName = Request.Query["BrandName"];
+            if (brandName == null || brandName.Length == 0)
+            {
+                int yearStarted = 2006;
```

Comment:
Can we store Year Started as a string, because we're unlikely to perform math on it.